winui/AvatarView/overview.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Overview of WinUI AvatarView control | Syncfusion
+description: Learn all about the introduction of Syncfusion WinUI AvatarView Control (SfAvatarView) with essential features and more here.
+platform: WinUI
+control: AvatarView
+documentation: ug
+---
+
+# WinUI AvatarView Overview
+
+The .NET AvatarView control provides a graphical representation of user image that allows you to customize the view by adding image, background color, icon, text, etc.
+
+![WinUI AvatarView control overview](avatarview_images/winui_avatarview_overview.png)
+
+## Key features
+
+* `Content type`: Display initials, images, and 25 pre-defined avatars.
+* `Group view`: Add up to three images or initials in a single view.
+* `Visual styles`: Use different types of visual styles such as circle, square and custom. 
+* `Customization`: Customize the height, width, border brush, border thickness, background, corner radius and font.
```

Comment:
* `Customization`: Customize the height, width, border brush, border thickness, background, corner radius, and font.